compiler/rustc_session/src/options.rs

```diff
@@ -1012,6 +1033,8 @@ options! {
         "divide crate into N units to optimize in parallel"),
     control_flow_guard: CFGuard = (CFGuard::Disabled, parse_cfguard, [TRACKED],
         "use Windows Control Flow Guard (default: no)"),
+    cf_protection: CFProtection = (CFProtection::None, parse_cfprotection, [TRACKED],
+        "Instrument control-flow architecture protection."),
```

Comment:
Could you also change this from a stable `-C` option to an unstable `-Z` option?